### PR TITLE
S21 alumni Graduated Class

### PR DESCRIPTION
**Pull request by roddy3**
Showing the graduating class for all alumni.   Issue #582

before 

Screen Shot 2021-07-07 at 10 43 43 AM

---------------------------------

After 
Screen Shot 2021-07-07 at 11 41 33 AM

### CODE REVIEW

File: src/services/user.js [hunk 1]
```diff
@@ -609,6 +609,7 @@ const getEmploymentInfo = async () => {
 
 const getProfileInfo = async (username) => {
   let profile = await getProfile(username);
+  console.log(profile);
```
Comment (josh-peters-2357) [inline src/services/user.js:612]:
```suggestion
```

File: src/views/PeopleSearch/components/PeopleSearchResult/index.js [hunk 2]
```diff
@@ -79,6 +79,9 @@ export default class PeopleSearchResult extends Component {
         case '7':
           personClassJobTitle = 'Graduate Conferred';
           break;
+        case '8':
+          personClassJobTitle = 'Class of ...';
+          break;
```
Comment (josh-peters-2357) [inline src/views/PeopleSearch/components/PeopleSearchResult/index.js:84]:
```suggestion
```

**Review by josh-peters-2357 — CHANGES_REQUESTED** [review]
Looks so good! I do have just a few suggestions.

File: src/views/PeopleSearch/components/PeopleSearchResult/index.js (hunk 2 above)
Comment (roddy3) [reply src/views/PeopleSearch/components/PeopleSearchResult/index.js]: Resolved!!!

File: src/services/user.js (hunk 1 above)
Comment (roddy3) [reply src/services/user.js]: Done!!!

**Review by Davideo37 — DISMISSED** [review]
LGTM! Nice work on this one!

**Review by silasw2 — DISMISSED** [review]
LGTM!

**Review by josh-peters-2357 — APPROVED** [review]

**Pull request merged**